### DISCUSSION TITLE
[Docker] How to deploy Budibase in a single container with databases outside of it?

### DISCUSSION THREAD

**Discussion by bfontaine** [Budibase/budibase#12562] [Help] [answered]
Hello,
I’m trying to deploy a Budibase instance with [Dokku](https://dokku.com/). From what I see, either I replicate all the containers from the [`docker-compose.yml`](https://raw.githubusercontent.com/Budibase/budibase/master/hosting/docker-compose.yaml) either I install a single container with [`budibase/budibase`](https://hub.docker.com/r/budibase/budibase) that contains everything.
Would there be a way to have the single container but still use external databases for Minio/Redis/CouchDB?
Dokku has plugins for [Redis](https://github.com/dokku/dokku-redis) and [CouchDB](https://github.com/dokku/dokku-couchdb), and I already have a working Minio installation. I prefer to have databases outside of the app container so I can better monitor and backup them.
Thanks

**Comment by mike12345567** [comment, marked as answer]
Hi @bfontaine - currently this is not possible, however it is worth noting that we do not use a standard CouchDB installation, without getting into the specifics it is necessary to utilise `budibase/couchdb` to run a CouchDB service that is compatible with Budibase.

If you wish to still attempt to run the services separately then we would recommend using a docker-compose setup, you will only need the proxy, app and worker services - we do not have an image that just does that.

**Reply by bfontaine** [reply]
Understood, thanks!
